packages/stainless/src/stl.ts

```diff
@@ -98,6 +134,10 @@ export interface BaseEndpoint<
 
 export type AnyBaseEndpoint = BaseEndpoint<any, any, any, any, any, any>;
 
+/**
+ * An endpoint created on an instance of {@link Stl} via the
+ * {@link Stl.endpoint} method.
```

Comment:
I don't normally use `{@link`… should I? Can you share a screenshot of what it does?

Reply:
<img width="473" alt="image" src="https://github.com/stainless-api/stl-api/assets/24882287/2700c045-e279-4d85-89a6-3d96ecada31f">
Clicking on the link navigates you to the relevant method. I've been a bit inconsistent in using `@link` though since it seems to require the types referenced to be imported, which isn't always the case, but I could just import them for documentation purposes. Or I could ditch the `@link` everywhere

Reply:
interesting. No that does look handy…

Reply:
My gut is to keep doing it where you feel like it, but not to become religious about doing it everywhere just yet